backend/config/autoload/frontend.global.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+return [
+    'frontend' => [
```

Comment:
Could you nest this in an 'ecamp' array, so we can use it in a similar way to the mail config etc.?

Reply:
done with fixup of backend/CampCollaborationService: send mail with created inviteKey